sim/mcuboot-sys/csupport/run.c

```diff
@@ -27,6 +27,13 @@
 
```

Comment:
`mbedtls_rsa_init` needs to be updated in line 151.

Reply:
Make sense.